test/Middleware/NavigationMiddlewareTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace ZendTest\Navigation\Middleware;
+
+use Interop\Http\ServerMiddleware\DelegateInterface;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ServerRequestInterface;
+use Zend\Expressive\Router\RouteResult;
+use Zend\Navigation\Exception\InvalidArgumentException;
+use Zend\Navigation\Middleware\NavigationMiddleware;
+use Zend\Navigation\Navigation;
+use Zend\Navigation\Page\ExpressivePage;
+
+class NavigationMiddlewareTest extends TestCase
+{
+    /**
+     * @var NavigationMiddleware
+     */
+    private $middleware;
+
+    /**
+     * @var Navigation
+     */
+    private $navigation;
+
+    protected function setUp()
+    {
+        $this->navigation = new Navigation(
+            [
```

Comment:
We can change formatting here to have:

```
$this->navigation = new Navigation([
    new ExpressivePage(['route' => 'home']),
]);
```

We do like that usually, to have less "almost empty" lines.